src/Resources/Resources/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 - Remove missing cmdlet referenced in `New-AzResourceGroupDeployment` documentation
```

Comment:
@markcowl Should I need to move the existing content in Upcoming release to Version 1.7.0? Or is it something that PowerShell team update after updating the PS modules?

Reply:
Just puttign it in Upcoming Release is correct - we have automated versioning tools that update the version based on semver.